keras_cv/models/convnext.py

```diff
@@ -0,0 +1,493 @@
+        drop_path_rate=drop_path_rate,
+        layer_scale_init_value=layer_scale_init_value,
+        weights=weights,
+        input_tensor=input_tensor,
+        input_shape=input_shape,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        model_name=model_name,
+    )
+
+
+def ConvNeXtSmall(
+    include_rescaling,
+    include_top,
+    drop_path_rate,
+    layer_scale_init_value,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classes=None,
+    classifier_activation="softmax",
+    model_name="convnext_small",
+):
+    return ConvNeXt(
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        depths=MODEL_CONFIGS["small"]["depths"],
+        projection_dims=MODEL_CONFIGS["small"]["projection_dims"],
+        drop_path_rate=drop_path_rate,
+        layer_scale_init_value=layer_scale_init_value,
+        weights=weights,
+        input_tensor=input_tensor,
+        input_shape=input_shape,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        model_name=model_name,
+    )
+
+
+def ConvNeXtBase(
+    include_rescaling,
+    include_top,
+    drop_path_rate,
+    layer_scale_init_value,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classes=None,
+    classifier_activation="softmax",
+    model_name="convnext_base",
+):
+    return ConvNeXt(
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        depths=MODEL_CONFIGS["base"]["depths"],
+        projection_dims=MODEL_CONFIGS["base"]["projection_dims"],
+        drop_path_rate=drop_path_rate,
+        layer_scale_init_value=layer_scale_init_value,
+        weights=weights,
+        input_tensor=input_tensor,
+        input_shape=input_shape,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        model_name=model_name,
+    )
+
+
+def ConvNeXtLarge(
+    include_rescaling,
+    include_top,
+    drop_path_rate,
+    layer_scale_init_value,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classes=None,
+    classifier_activation="softmax",
+    model_name="convnext_large",
+):
+    return ConvNeXt(
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        depths=MODEL_CONFIGS["large"]["depths"],
+        projection_dims=MODEL_CONFIGS["large"]["projection_dims"],
+        drop_path_rate=drop_path_rate,
+        layer_scale_init_value=layer_scale_init_value,
+        weights=weights,
+        input_tensor=input_tensor,
+        input_shape=input_shape,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        model_name=model_name,
+    )
+
+
+def ConvNeXtXLarge(
```

Comment:
Please add Docstrings for these exported methods using the convention from the DenseNet impl:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/densenet.py

Reply:
Such an awesome piece of work that is!